### PR TITLE
Issue ck 482

### PR DESCRIPTION
**Pull request by lowlandjuju**
Adds a cookie consent popup for GDPR compliance. 

## Description
- Uses this package: https://github.com/orestbida/cookieconsent
- The cookie consent manages its own consent cookie, plus two cookies from PostHog 

## Motivation and Context
Fixes issue CK-482 (Common Knowledge)

## How Can It Be Tested?
Manually

### CODE REVIEW

File: smartforests/templates/cookie_consent.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                                  title: 'Strictly necessary cookies',
+                                  description: 'These cookies are essential for the proper functioning of my website. Without these cookies, the website would not work properly',
+                                  toggle: {
+                                      value: 'necessary',
+                                      enabled: true,
+                                      readonly: true           
+                                  }
+                              }, {
+                                  title: 'Performance and Analytics cookies',
+                                  description: 'These cookies allow the website to remember the choices you have made in the past',
```
Comment (janbaykara) [inline smartforests/templates/cookie_consent.html:56]: This isn't quite true. These cookies allow the research team to collect usage data so they can improve the website over time.

**Review by janbaykara — CHANGES_REQUESTED** [review]
This is very nice and neat. I've not tested it but assume you have! Have requested one quick change to the copy, to be a bit more accurate.

File: smartforests/templates/cookie_consent.html (hunk 1 above)
Comment (lowlandjuju) [reply smartforests/templates/cookie_consent.html]: Thanks @janbaykara :pray:  - this is resolved now

**Review by janbaykara — APPROVED** [review]
Ship it!

**Pull request merged**